webhooks/services/acapy_events_processor.py

```diff
@@ -338,6 +340,38 @@ def _process_event(self, event_json: str) -> None:
                 event_json=webhook_event_json, transaction_id=transaction_id
             )
 
+        if wallet_id != GOVERNANCE_LABEL and (
+            is_applicable_for_billing(
+                topic=cloudapi_topic, payload=payload, logger=bound_logger
+            )
+        ):
+            bound_logger.info(
+                "Forwarding billing event for Billing service",
+            )
+            if cloudapi_topic == "endorsements":
+                # Add the operation type to the payload for endorsements
+                # Simplifies the billing service's logic for determining the operation type
+                endorse_event: Dict[str, Any] = json.loads(webhook_event_json)
```

Comment:
But, we should avoid serialising / deserialising if possible. 

You're essentially modifying the payload to include the "type" key with the operation_type (which I presume is because operation type is nested in a json string in message_attach: data, etc. 
We can do that without loading the json.

Earlier we had:
```
webhook_event_json = cloudapi_webhook_event.model_dump_json()
```

You can copy the `cloudapi_webhook_event`, replace payload directly (because already have the payload dict in scope). So, payload["type"] = operation_type, which is fine because payload is not used elsewhere again. Then `modified_model = cloudapi_webhook_event.copy(payload = payload)` <- something like that, Then 
`webhook_event_for_billing = modified_model.model_dump_json()` <- use pydantic's method to do it.

So, we then just have 1 more json operation, instead of 2

Then to deduplicate the .add_billing_event call (like mentioned in other comment), you can just say:
```
            if cloudapi_topic == "endorsements":
...
                webhook_event_for_billing = modified_model.model_dump_json()

            else:
                webhook_event_for_billing = webhook_event_json

            self.redis_service.add_billing_event(
                event_json=webhook_event_for_billing ,
                group_id=group_id,
                wallet_id=wallet_id,
                timestamp_ns=event.metadata.time_ns,
            )